Code/Editor/QtViewPaneManager.cpp

```diff
@@ -328,6 +330,15 @@ bool DockWidget::event(QEvent* qtEvent)
         reparentToMainWindowFix();
     }
 
+    if (qtEvent->type() == QEvent::Close)
+    {
+        QTimer::singleShot(0, this, [pane = m_pane]()
```

Comment:
a comment as to why this can't be done synchronously, please :)

Reply:
Added a comment! A frame is needed for the pane's state to be updated. Else the notification would trigger, but other systems querying the state of the pane would still see it as active.